README.md

```diff
@@ -114,3 +114,136 @@ spec:
 EOF
 ```
 
+## Install in modular kyma on local k3d cluster
+
+1. Setup local k3d cluster and local docker registry
+
+```bash
+k3d cluster create kyma --registry-create registry.localhost:0.0.0.0:5001
+```
+2. Add `etc/hosts` entry to register the local docker registry under a `registry.localhost` name
+
+```
+127.0.0.1 registry.localhost
+```
+
+3. Export ENVs pointing to module and module image registries
+
+```bash
+export IMG_REGISTRY=registry.localhost:5001/unsigned/operator-images
+export MODULE_REGISTRY=registry.localhost:5001/unsigned
+```
+
+4. Build Keda module
+```bash
+make module-build
+```
+
+This will build oci image for keda module and push it to the registry and path as defined in `MODULE_REGISTRY`.
+
+5. Build Keda manager image
+```bash
+make module-image
+```
+
+This will build docker image for keda manager and push it to the registry and path as defined in `IMG_REGISTRY`.
+
+6. Verify if the module and the manager's image are pushed to the local registry
+
+```bash
+curl registry.localhost:5001/v2/_catalog
+{"repositories":["unsigned/component-descriptors/kyma.project.io/module/keda","unsigned/operator-images/keda-operator"]}
+```
+
+7. Inpect the generated module template
+
+The following are temporary workarounds. Probabaly will be mitigated in the future.
+
+Edit the `template.yaml` file and:
+
+ - change the `target` to `control-plane`
+This is only required in the single cluster mode
+
+```yaml
+spec:
+  target: control-plane
+```
+
+- change the existing repository context in spec.descriptor.component:
+(Because pods inside the k3d cluster use the docker-internal port of the registry, it will try to resolve the registry against port 5000 instead of 5001. K3d has registry aliases but module-manager is not part of k3s and thus does not know how to properly alias `registry.localhost:5001`)
```

Comment:
```suggestion
>**NOTE:** Because Pods inside the k3d cluster use the docker-internal port of the registry, it tries to resolve the registry against port 5000 instead of 5001. K3d has registry aliases but module-manager is not part of k3d and thus does not know how to properly alias `registry.localhost:5001`
```